### DISCUSSION TITLE
How to deconstruct, modernize, and modularize a massive query

### DISCUSSION THREAD

**Discussion by lbenezra-FA** [dbt-labs/docs.getdbt.com#1757] [Archive]
Suggested by @jasnonaz ! blog about deconstructing a massive > 2500 line SQL query into readable, modular queries using dbt best design practices.

**Comment by jasnonaz** [comment]
Yess thank you for opening this up Lauren! Think this will be a stellar post - lets go ahead and [open an issue](https://github.com/dbt-labs/docs.getdbt.com/issues/new?assignees=&labels=content%2Cdeveloper+blog&template=contribute-to-developer-blog.yml) and we can get this moving.

**Reply by lbenezra-FA** [reply]
done!
